processor/src/main/java/com/log/processor/LogProcessorModule.java

```diff
@@ -44,12 +46,15 @@
 	public LogProcessorModule(LogProcessorConfiguration c, Environment e) {
 		this.configuration = c;
 		this.environment = e;
+		System.out.println("control here - 1");
```

Comment:
remove System out messages. Do not check in these. For debugging use log messages not system outs